src/Arcus.Security.Providers.AzureKeyVault/Authentication/ManagedServiceIdentityAuthenticator.cs

```diff
@@ -8,12 +8,24 @@ namespace Arcus.Security.Providers.AzureKeyVault.Authentication
     /// <summary>
     ///     Azure Key Vault authentication by using Azure Managed Service Identity
     /// </summary>
-    public class ManagedServiceIdentityAuthenticator : IKeyVaultAuthenticator
+#pragma warning disable 618
+    public class ManagedServiceIdentityAuthenticator : IKeyVaultAuthentication, IKeyVaultAuthenticator
+#pragma warning restore 618
     {
         /// <summary>
         /// Authenticates with Azure Key Vault
         /// </summary>
         /// <returns>A <see cref="KeyVaultClient"/> client to use for interaction with the vault</returns>
```

Comment:
```suggestion
        /// <returns>A <see cref="IKeyVaultClient"/> client to use for interaction with the vault</returns>
```

Reply:
Was going to merge this one but I can't, feel free to commit this suggestion if you agree @stijnmoreels